src/DB/Dump.php

```diff
@@ -36,19 +35,20 @@ public function __construct(
      */
     public function getCommand(): string
     {
-        $command = 'mysqldump -h ' . escapeshellarg($this->connectionData->getHost())
-            . ' -u ' . escapeshellarg($this->connectionData->getUser());
+        $connectionData = $this->connectionFactory->create(ConnectionFactory::CONNECTION_SLAVE);
```

Comment:
Don't we want to share a connection with `get()`?

Reply:
Sorry, don't udnerstand the question. Can you clarify please?

Reply:
Like `make` always create a new instance of connection i.e. opens a new connection. `get` would share this as singleton without creating a new one if it's already exists

Reply:
The configuration actually cached in MergedConfig. So there no additional opertion for creating new instance of Connection.